packages/web/components/Link.tsx

```diff
@@ -1,13 +1,18 @@
-import { Link } from '@metafam/ds';
+import { Link, LinkProps as DSLinkProps } from '@metafam/ds';
 import NextLink, { LinkProps } from 'next/link';
 import React from 'react';
 
-type Props = LinkProps;
+type MetaLinkProps = LinkProps & DSLinkProps;
```

Comment:
The reason you are having type errors is that DSLinkProps is overwriting the "as" prop in LinkProps to something else. There's other conflicting types too in those objects, and if you just join them like this it thinks you want to accept either of these props. 

Instead, we should only use the DSLinkProps that aren't already defined in LinkProps. We can do that like this:
```suggestion
type MetaLinkProps = Omit<DSLinkProps, keyof LinkProps> & LinkProps;
```

This "omits" any of the props in DSLinkProps that match the name of any props in LinkProps

Reply:
Thanks you!